tests/diagramconfiguration_test.py

```diff
@@ -250,24 +329,38 @@ def test__reveal_children__no_duplicates_after_multiple_reveal(directory):
 
     assert child_tree_item.childCount() == count
 
-def test__reveal_children__check_if_reveals__current_item_is_null(directory):
+
+def test__diagram_configuration__reveal_already_revealed_item__no_action(directory):
+    diagram_configuration = setup_diagram_configuration(directory)
+
+    entry_point_item = diagram_configuration.entry_point_item_
+    diagram_configuration.tree_.setCurrentItem(entry_point_item)
+
+    with patch.object(entry_point_item, 'get_callable') as mock:
+        diagram_configuration.revealChildren()
+
+    mock.assert_not_called()
+
+
+def test__diagram_configuration__reveal_if_no_item_selected__no_action(directory):
```

Comment:
Consider to use the same test approach for tests `test__diagram_configuration__reveal_already_revealed_item__no_action` and `test__diagram_configuration__reveal_if_no_item_selected__no_action` as both are supposed to trigger no action

Reply:
done